examples/tutorial_client_eventfilter.c

```diff
@@ -0,0 +1,267 @@
+    contentFilter[0].elements[1].filterOperands[0].content.decoded.data = literalOperand;
+    /* third Element (OfType) */
+    contentFilter[0].elements[2].filterOperands[0].content.decoded.type = &UA_TYPES[UA_TYPES_LITERALOPERAND];
+    contentFilter[0].elements[2].filterOperands[0].encoding = UA_EXTENSIONOBJECT_DECODED;
+    UA_LiteralOperand *secondLiteralOperand = UA_LiteralOperand_new();
+    UA_LiteralOperand_init(secondLiteralOperand);
+    UA_NodeId *secondNodeId = UA_NodeId_new();
+    UA_NodeId_init(secondNodeId);
+    secondNodeId->namespaceIndex = 0;
+    secondNodeId->identifierType = UA_NODEIDTYPE_NUMERIC;
+    secondNodeId->identifier.numeric = UA_NS0ID_EVENTQUEUEOVERFLOWEVENTTYPE;
+    UA_Variant_setScalar(&secondLiteralOperand->value, secondNodeId, &UA_TYPES[UA_TYPES_NODEID]);
+    contentFilter[0].elements[2].filterOperands[0].content.decoded.data = secondLiteralOperand;
+    return contentFilter;
+}
+
+static UA_SimpleAttributeOperand *
+setupSelectClauses(void) {
+    UA_SimpleAttributeOperand *selectClauses = (UA_SimpleAttributeOperand*)
+        UA_Array_new(nSelectClauses, &UA_TYPES[UA_TYPES_SIMPLEATTRIBUTEOPERAND]);
+    if(!selectClauses)
+        return NULL;
+    for(size_t i =0; i<nSelectClauses; ++i) {
+        UA_SimpleAttributeOperand_init(&selectClauses[i]);
+    }
+    selectClauses[0].typeDefinitionId = UA_NODEID_NUMERIC(0, UA_NS0ID_BASEEVENTTYPE);
+    selectClauses[0].browsePathSize = 1;
+    selectClauses[0].browsePath = (UA_QualifiedName*)
+        UA_Array_new(selectClauses[0].browsePathSize, &UA_TYPES[UA_TYPES_QUALIFIEDNAME]);
+    if(!selectClauses[0].browsePath) {
+        UA_SimpleAttributeOperand_delete(selectClauses);
+        return NULL;
+    }
+    selectClauses[0].attributeId = UA_ATTRIBUTEID_VALUE;
+    selectClauses[0].browsePath[0] = UA_QUALIFIEDNAME_ALLOC(0, "Message");
+    selectClauses[1].typeDefinitionId = UA_NODEID_NUMERIC(0, UA_NS0ID_BASEEVENTTYPE);
+    selectClauses[1].browsePathSize = 1;
+    selectClauses[1].browsePath = (UA_QualifiedName*)
+        UA_Array_new(selectClauses[1].browsePathSize, &UA_TYPES[UA_TYPES_QUALIFIEDNAME]);
+    if(!selectClauses[1].browsePath) {
+        UA_SimpleAttributeOperand_delete(selectClauses);
+        return NULL;
+    }
+    selectClauses[1].attributeId = UA_ATTRIBUTEID_VALUE;
+    selectClauses[1].browsePath[0] = UA_QUALIFIEDNAME_ALLOC(0, "Severity");
+
+    return selectClauses;
+}
+#endif
+
+int main(int argc, char *argv[]) {
+    signal(SIGINT, stopHandler);
+    signal(SIGTERM, stopHandler);
+
+    if(argc < 1) {
+        printf("Usage: tutorial_client_events <opc.tcp://server-url>\n");
+        return EXIT_FAILURE;
+    }
+
+    UA_Client *client = UA_Client_new();
+    UA_ClientConfig_setDefault(UA_Client_getConfig(client));
+
+    /* opc.tcp://uademo.prosysopc.com:53530/OPCUA/SimulationServer */
+    /* opc.tcp://opcua.demo-this.com:51210/UA/SampleServer */
+    UA_StatusCode retval = UA_Client_connect(client, "opc.tcp://ammar:4840/");
```

Comment:
Please remove "ammar" here and use e.g. localhost as the default. Remove the example server since this example should be used together with the event filter server example.

Reply:
We overlooked "Ammar" when we went through the code. The host is now passed as an argument similar to the other tutorials.